lib/ruby_audit/database.rb

```diff
@@ -10,12 +10,12 @@ def advisories_for(name, type)
       end
     end
 
-    def check_ruby(ruby, &block)
-      check(ruby, 'rubies', &block)
+    def check_ruby(ruby, &)
+      check(ruby, 'rubies', &)
```

Comment:
Rubocop [encourages](https://docs.rubocop.org/rubocop/cops_naming.html#namingblockforwarding) using "anonymous block forwarding", which was introduced in Ruby 3.1.